### DISCUSSION TITLE
Rename /oam to /patterns or to /capabilities

### DISCUSSION THREAD

**Discussion by leecalcote** [meshery/meshery#3975] [General]
@utkarsh-pro, what do you think? https://github.com/meshery/meshery/tree/master/oam --> /patterns or /capabilities or move this folder to https://github.com/meshery/meshery/tree/master/models/pattern/patterns?

**Comment by tangledbytes** [comment]
YES, please. I think renaming `oam` to `capabilities` would be a bit better. Renaming it to `patterns` may be a bit deceptive, as they are not actual _patterns_. 

Moving it inside `models/pattern/patterns` directory would be slightly inconvenient from code point of view and also that directory currently holds the implementation of core meshery resources.

**Reply by leecalcote** [reply]
`capabilities` sounds great. Is there another location other than root that might be equally as convenient?
